1_callbacks.js

```diff
@@ -2,23 +2,35 @@ const fs = require('fs');
 
 let finalDoc = [];
 
-fs.readFile('data/p1.txt', 'utf8', (err, data) => {
-  console.log(data);
+const done = function (finalResults) {
+  console.log('ALL DONE');
+  console.log(finalResults.toString().replace(/,/g, "\n"));
```

Comment:
Why not `.join('\n')`?

Reply:
That's one step less so even better. I didn't think of .join

Reply:
console.log(finalResults.join('\n');